gr-pdu/lib/time_delta_impl.cc

```diff
@@ -84,7 +82,9 @@ void time_delta_impl::handle_pdu(const pmt::pmt_t& pdu)
 
     gr::thread::scoped_lock l(d_mutex);
 
-    double t_now((boost::get_system_time() - d_epoch).total_microseconds() / 1000000.0);
+    double t_now(
```

Comment:
Same comment as above. Also, I wonder if seconds are fine enough resolution here (unrelated to your PR) and if we should have a separate time-helper header with code like this (getting the number of secs since epoch seems useful elsewhere, too)

Reply:
> I wonder if seconds are fine enough resolution here

The time is represented as a `double` so the resolution should be on the order of a microsecond.

Reply:
OK, I thought that `count()` would literally return the number of tick counts, and always return an integer (and then that gets cast to double). But that's not the case here (also it would break previous functionality... so.... yeah).